x-pack/legacy/plugins/graph/public/components/app.tsx

```diff
@@ -45,14 +45,9 @@ export function GraphApp(props: GraphAppProps) {
         <Provider store={reduxStore}>
           <>
             <div className="gphGraph__bar">
-              <EuiFlexGroup direction="column" gutterSize="s">
-                <EuiFlexItem>
-                  <SearchBar {...searchBarProps} />
-                </EuiFlexItem>
-                <EuiFlexItem>
-                  <FieldManager pickerOpen={pickerOpen} setPickerOpen={setPickerOpen} />
-                </EuiFlexItem>
-              </EuiFlexGroup>
+              <SearchBar {...searchBarProps} />
```

Comment:
Using a spacer instead of a vertical flex group because IE is not able to layout them correctly